### PR TITLE
Change package name to com.azure.xxx for Spring Boot modules

### PR DESCRIPTION
**Pull request by wujack778**
Update package name from com.microsoft.azure to com.azure for track 2 library.
 [issue 15765](https://github.com/Azure/azure-sdk-for-java/issues/15765)

### CODE REVIEW

**Comment by saragluna** [comment]
Please resolve conflicts.

**Comment by wujack778** [comment]
> Please resolve conflicts.

fine

**Review by moarychan — APPROVED** [review]
LGTM. Need to update artifact id and parameters linked to Maven in readme documents, such as  `sdk/spring/README.md`, `sdk/spring/azure-spring-boot-starter/README.md`.

**Review by backwind1233 — COMMENTED** [review]
some blank lines

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/keyvault/spring/KeyVaultEnvironmentPostProcessor.java [hunk 1]
```diff
@@ -50,29 +50,28 @@ public void postProcessEnvironment(ConfigurableEnvironment environment, SpringAp
...
      * @return true if the key vault is enabled, false otherwise.
      */
     private boolean isKeyVaultEnabled(ConfigurableEnvironment environment, String normalizedName) {
         return environment.getProperty(
-                KeyVaultProperties.getPropertyName(normalizedName, Property.ENABLED),
+                KeyVaultProperties.getPropertyName(normalizedName, KeyVaultProperties.Property.ENABLED),
                 Boolean.class,
                 true)
-            && environment.getProperty(KeyVaultProperties.getPropertyName(normalizedName, Property.URI)) != null
+            && environment.getProperty(KeyVaultProperties
```
[the remark this answers is not in the document]
Comment (wujack778) [reply sdk/spring/azure-spring-boot/src/main/java/com/azure/keyvault/spring/KeyVaultEnvironmentPostProcessor.java:63]: this is a linelength check there.

File: sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/aad/AADAuthenticationAutoConfigurationTest.java [hunk 2]
```diff
@@ -54,7 +55,7 @@ public void serviceEndpointsCanBeOverridden() {
                 assertThat(serviceEndpointsProperties.getEndpoints()).isNotEmpty();
 
                 final Map<String, ServiceEndpoints> endpoints = serviceEndpointsProperties.getEndpoints();
-                assertThat(endpoints).hasSize(4);
+                Assertions.assertThat(endpoints).hasSize(4);
```
[the remark this answers is not in the document]
Comment (wujack778) [reply sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/aad/AADAuthenticationAutoConfigurationTest.java:58]: done

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java [hunk 3]
```diff
@@ -21,13 +20,14 @@
...
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosHealthIndicator.class);
+
     @Value("${azure.cosmos.database}")
     private String dbName;
+
     @Value("${azure.cosmos.uri}")
     private String uri;
 
-    @Autowired
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java:29]: Does the @Autowired annotation can be removed here?

**Review by backwind1233 — APPROVED** [review]

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java (hunk 3 above)
Comment (saragluna) [reply sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java:29]: Yes, the field is initialized from the constructor.

**Pull request merged**